src/Data/Parity/Base.agda

```diff
@@ -0,0 +1,106 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Parity
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Parity.Base where
+
+open import Algebra.Bundles.Raw using (RawMagma; RawMonoid; RawNearSemiring; RawSemiring)
+open import Data.Sign.Base using (Sign; +; -)
+open import Level using (0ℓ)
+open import Relation.Binary.PropositionalEquality
+
+------------------------------------------------------------------------
+-- Definition
+
+data Parity : Set where
+  0ℙ : Parity
+  1ℙ : Parity
+
+------------------------------------------------------------------------
+-- Operations
+
+-- The opposite parity.
+
+_ᵒ : Parity → Parity
```

Comment:
I'm really sorry, but I'm not sure about this new `_ᵒ` name... I suggested `-_` because it has a clear well-defined semantics as "negate" and therefore users can work out what "negating" a parity does. However `_ᵒ` lacks any such common semantics. If you're not keen on `-_` then I'd much prefer we go back to `opposite`?

Reply:
Well, the symbol is 'well-defined' in some contexts... and I shared @Taneb 's aversion to prefix unary negation. 

I think if this choice doesn't fly, I think I'd rather/better go for _^-1